### PR TITLE
Run full e2e tests on TravisCI and build docker

### PR DESCRIPTION
**Pull request by SergKam**
The goal is to fully test Nakadi UI in isolation, build and publish docker that can be used as a base layer in the real deployment.

### CODE REVIEW

**Comment by SergKam** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
   - elm-format --validate .
-  - npm run test:client
+  - npm run test:ci
+  - docker build -t "nakadi/nakadi-ui:${TRAVIS_BRANCH}-${TRAVIS_BUILD_NUMBER}" .
+  - docker tag "nakadi/nakadi-ui:${TRAVIS_BRANCH}-${TRAVIS_BUILD_NUMBER}" nakadi/nakadi-ui:latest
+
+deploy:
+  provider: script
+  script:
+    echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin && bash docker push nakadi/nakadi-ui
```
Comment (adyach) [inline .travis.yml]: `bash docker push nakadi/nakadi-ui` why do you need `bash` here ?

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
     - $HOME/.npm # https://stackoverflow.com/a/42523517/1148249
 
+before_install:
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+
 script:
   - elm-format --validate .
-  - npm run test:client
+  - npm run test:ci
+  - docker build -t "nakadi/nakadi-ui:${TRAVIS_BRANCH}-${TRAVIS_BUILD_NUMBER}" .
```
Comment (adyach) [inline .travis.yml]: does Nakadi UI support compatibility with Nakadi in anyway at the moment?

**Comment by adyach** [comment]
@SergKam do you really want to build docker image on every commit to master?

File: .travis.yml (hunk 1 above)
Comment (SergKam) [reply .travis.yml]: :man_facepalming: copy pasting

File: .travis.yml (hunk 2 above)
Comment (SergKam) [reply .travis.yml]: Nakadi API stable enough so I don't see the need for versions synchronization for now.
But we can add it in the future.

Comment (adyach) [reply .travis.yml]: I am fine with it :) just wanted to highlight that you have to use always latest version of Nakadi UI and Nakadi together

**Comment by SergKam** [comment]
Yes, why not? It is easy. we can distinguish between stable and dev versions. But again let's start simple. Every merge will be tested and published so users can use fresh features immediately.

**Comment by adyach** [comment]
👍

**Comment by SergKam** [comment]
:+1:

**Pull request merged**